pages_/my-shop.tsx

```diff
@@ -257,19 +268,23 @@ const MyShop = ({ isLoggedIn, shopId }: Props) => {
       ) : null}
 
       <Card className="cauda_card cauda_card--clean mb-4 mx-auto">
-        <Card.Body className="p-0 text-center d-flex justify-content-center align-items-center">
+        <Card.Body className="p-0 text-center d-flex flex-column justify-content-center align-items-center">
           <Link href="/shop-poster" passHref>
             <Button
               as="a"
               target="blank"
-              variant="primary"
+              variant="outline-primary"
               className="d-flex justify-content-between align-items-center py-2"
             >
               <FontAwesomeIcon icon={faPrint} fixedWidth className="mr-2" />
               {t('common:print-poster')}
               <div></div>
             </Button>
           </Link>
+
+          <p className="mt-2 mb-0">
+            <small>https://cauda.app/{encodeId(shopId)}</small>
```

Comment:
@nicosampler the shopId encoding should be done in the getServerSideProps function before sending it as props to the component so the hashids lib wouldn't be added to the browser bundle